pkg/apis/nfs.rook.io/v1alpha1/register.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	nfsrookio "github.com/rook/rook/pkg/apis/nfs.rook.io"
+)
+
+const (
+	CustomResourceGroup = "nfs.rook.io"
+	Version             = "v1alpha1"
```

Comment:
Are you using a golang formatting tool? In rook we typically use `goreturns` to format our whitespace consistently.

Reply:
I used `go fmt` , will use `goreturns` from now on

Reply:
I used `goreturns` and output is same

Reply:
ok thanks, looks like some whitespace changed